.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Testing here, it says:
> hint: The '.husky/pre-commit' hook was ignored because it's not set as executable.

Try `chmod +x .husky/pre-commit`
and commit again

Reply:
Done.

Reply:
Not sure why, but it didn't worked for me,
```txt
[nix-shell:~/Code/tmp/laconia]$ ls -l .husky/pre-commit
-rw-r--r-- 1 hugosenari users 57 fev 14 09:33 .husky/pre-commit
```

Could you please clone your repository in other dir, change to this branch and test a commit?

Reply:
Sure.

Reply:
It's not working, I am unable to figure out how to fix it.
Please help me.

Reply:
Did you run npm run prepare after checkout?
(Just to be sure)

Reply:
Marking it as resolved,
Please fill a new issue with this error and your environment details